### PR TITLE
Scrambled words

### PR DESCRIPTION
**Pull request by jyothsnashaji**
### **Describe your change:**
Added an algorithm to unscrambled a word into its meaningful English spelling

* [x ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: strings/unscramble.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        raise ValueError("NOT-A-STRING")
+
+    scrambled_words = [''.join(i) for i in permutations(test_word)]
+    solutions = []
+
+    for word in set(scrambled_words):
+        if word in dictionary :
+            solutions.append(word)
+
+    return set(solutions)
```
Comment (L3str4nge) [inline strings/unscramble.py:47]: Instead of calling set we can just define solutions as a set: `solutions = set()`

File: strings/unscramble.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    ValueError: NOT-A-STRING
+    """
+
+    if not test_word.isalpha():
+        raise ValueError("NOT-A-STRING")
+
+    scrambled_words = [''.join(i) for i in permutations(test_word)]
+    solutions = []
+
+    for word in set(scrambled_words):
```
Comment (L3str4nge) [inline strings/unscramble.py:43]: It could be pythonic one liner

**Review by L3str4nge — CHANGES_REQUESTED** [review]

**Comment by jyothsnashaji** [comment]
@L3str4nge  Can you please review and close?

**Comment by cclauss** [comment]
In preparation for that denial-of-service attack that is [Hacktoberfest](https://hacktoberfest.com), I am closing all pull requests that do not have passing tests.

**Pull request closed without merging**